src-java/kilda-pce/src/test/java/org/openkilda/pce/impl/MaxLatencyPathComputationStrategyBaseTest.java

```diff
@@ -273,6 +280,26 @@ public void maxLatencyShouldChooseCorrectWayTest() throws Exception {
         assertEquals(path.getForward().getSegments().get(1).getSrcSwitchId(), new SwitchId("00:03"));
     }
 
+    @Test
+    public void shouldNotFindPathsGreaterThenMaxLatency() throws Exception {
```

Comment:
Please also add a test for the maxLatencyTier2 case.

Reply:
Done